electionleaflets/templates/leaflets/leaflet.html

```diff
@@ -1,156 +1,124 @@
+                </a>
+            </div>
+        </figure>
+    {% endfor %}
+
+    <h2>Leaflet details</h2>
+    <dl class="ds-descriptions">
+        <div>
+            <dt>Uploaded</dt>
+            <dd>{{ object.date_uploaded|date:"l, d M, Y" }}</dd>
+        </div>
+        {% if object.nuts1 %}
+            <div>
+                <dt>Region</dt>
+                <dd><a href="{% url 'leaflets' %}?filter_by_region={{ object.nuts1 }}">{{ object.nuts1_name }}</a></dd>
+            </div>
+        {% endif %}
+
+        {% if object.ynr_party_id %}
+            <div>
+                <dt>Published by</dt>
+                <dd><a href="{% url "party-view" object.ynr_party_id %}">{{ object.ynr_party_name }}</a></dd>
+            </div>
+        {% endif %}
+        {% if object.ballots %}
+
+
+            <div>
+                <dt>Related elections</dt>
+                <dd>
+                    <ul>
+                        {% for ballot in object.ballots %}
+                            <li>
+                                <a href="https://whocanivotefor.co.uk/elections/{{ ballot.ballot_paper_id }}">{{ ballot.ballot_title }}</a>
+                            </li>
+                        {% endfor %}
+                    </ul>
+                </dd>
+            </div>
         {% endif %}
-      {% endfor %}
-    </ul>
-  {% endif %}
-
-
-
-  <h2>Leaflet details</h2>
-  <dl class="ds-descriptions">
-    <div>
-      <dt>Uploaded</dt>
-      <dd>{{ object.date_uploaded|date:"l, d M, Y" }}</dd>
-    </div>
-    {% if object.nuts1 %}
-    <div>
-      <dt>Region</dt>
-        <dd><a href="{% url 'leaflets' %}?filter_by_region={{ object.nuts1 }}">{{ object.nuts1_name }}</a></dd>
-    </div>
-    {% endif %}
 
-    {% if object.ynr_party_id %}
-    <div>
-      <dt>Published by</dt>
-      <dd><a href="{% url "party-view" object.ynr_party_id %}">{{ object.ynr_party_name }}</a></dd>
-    </div>
-    {% endif %}
-    {% if object.ballots %}
-
-
-      <div>
-        <dt>Related elections</dt>
-        <dd>
-          <ul>
-            {% for ballot in object.ballots %}
-              <li><a href="https://whocanivotefor.co.uk/elections/{{ ballot.ballot_paper_id }}">{{ ballot.ballot_title }}</a></li>
-            {% endfor %}
-          </ul>
-        </dd>
-      </div>
-    {% endif %}
 
 
+        {% if object.people %}
+            <div>
+                <dt>Sent by</dt>
+                <dd>
+                    <ul style="list-style-type: none; padding-left:0 ">
+                        {% for id, person in object.people.items %}
+                            <li><a href="{% url "person" person.person.id %}">{{ person.person.name }}</a></li>
+                        {% endfor %}
+                    </ul>
+                </dd>
+            </div>
+
+        {% endif %}
+    </dl>
+
+    <p>
+        The data on this page is available in <a href="{% url "api:leaflet-detail" object.pk %}?format=json">JSON
```

Comment:
Tangent, but noticed it while testing these PRs.
As far as I can see, this link returns `{"detail":"No Leaflet matches the given query."}` for all leaflets. If it is not a quick job to fix it, shall we just remove this link?